Gemfile

```diff
@@ -25,6 +25,14 @@ group(:development, :test) do
   gem 'cheffish', "~> 1.3", "!= 1.3.1"
 
   gem 'ruby-shadow', :platforms => :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+
+  # For external tests
+  gem 'chefspec', github: 'jkeiser/chefspec', branch: 'jk/chefspec-12.5'
+  gem 'chef-sugar'
+  gem 'poise', github: 'poise/poise', branch: 'deeecb890a6a0bc2037dfb09ce0fd0a8931519aa'
```

Comment:
Ahh, hmm. Maybe replace `__FILE__` in the munger with a path equivalent to where the gemfile would be normally?